mindsdb/api/mysql/mysql_proxy/datahub/datanodes/mindsdb_datanode.py

```diff
@@ -265,6 +266,13 @@ def select(self, table, columns=None, where=None, where_data=None, order_by=None
             return data
         else:
             pred_dicts, explanations = self.model_interface.predict(table, 'dict&explain', when_data=where_data)
+            # Fix since for some databases we *MUST* return the same value for the columns originally specified in the `WHERE`
+            if isinstance(where_data, list):
```

Comment:
Im think `where_data` can be `dict` in some cases. Should we process that case?

Reply:
Oh, ok